configure_polaris_envs.py

```diff
@@ -101,14 +46,16 @@ def main():
     install_miniforge(conda_base, activate_base, logger)
 
     local_mache = args.mache_fork is not None and args.mache_branch is not None
-    if local_mache:
-        mache = ''
-    else:
+
+    packages = '--file deploy/spec-bootstrap.txt'
+    if not local_mache:
+        # we need to specify the mache version and all dependencies
+        # since we won't be installing from mache's spec file
```

Comment:
Maybe this would be a little more accurate?
```suggestion
        # we need to add the mache package, specifying a version,
        # since we won't be installing mache from a local clone of a branch
```
The dependencies for `mache` are coming from the conda package for `mache`, so we're not specifying them here.  What's coming from `deploy/spec-bootstrap.txt` are dependencies for `bootstrap.py` other than `mache`.  They don't relate to `mache` itself.

Reply:
Once this comment is updated (with something along the lines of my suggestions), feel free to merge the PR.